src/demo/pages/DemoContainer.jsx

```diff
@@ -69,9 +69,24 @@ const loggerClick = () => console.log('click'); // eslint-disable-line no-consol
 const submitter = () => console.log('submitted'); // eslint-disable-line no-console
 
 class DemoContainer extends React.Component {
+  static generateRandomString() {
+    const texts = [
+      'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
```

Comment:
Lets please use longer texts. It takes a lot of clicks to fill up the `ScrollView` area for vertical scroll on bigger screens.